### PR TITLE
Feature/pack voting

### PR DESCRIPTION
**Pull request by Ryan-D-Miller**
##### Description
What's changed & where should the reviewer start?
Added the pack voting page and functionality to add upvoted names to the pack. currently doesnt persist in any way and is only looking at the user id of 1 

#### Type of change
What is it?
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring

### Checklist:
- [x] My code follows the guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream module

### CODE REVIEW

File: src/Components/Pack/Pack.js [hunk 1]
```diff
@@ -1,21 +1,59 @@
...
+            pack: {},
+            names: data.names,
+            currentName: 0,
+            votingDone: false
+        }
+    }
+
+    componentDidMount() {
+        const id = parseInt(this.props.match.params.id);
+        const setPack = packData.packs.find(pack => pack.id === id);
```
Comment (EllieAzaveda) [inline src/Components/Pack/Pack.js:20]: Thanks for adding the mock data for now! This looks good!

File: src/Components/Pack/Pack.js [hunk 2]
```diff
@@ -1,21 +1,59 @@
...
 
     render() {
         return (
             <main className="pack">
-                <p>hello</p>
+                {!this.state.votingDone && <section className="vote-block"><button onClick={() => this.downVote()}>Down Vote!</button>
+                <section>
+                    <p>{this.state.names[this.state.currentName].name}</p>
+                </section>
+                <button onClick={() => this.upVote()}>Up Vote!</button></section> }
```
Comment (EllieJellyBean) [inline src/Components/Pack/Pack.js:52]: I'll stick to the down vote and up vote naming conventions too. Good naming choice!

File: src/Components/Pack/Pack.js [hunk 3]
```diff
@@ -1,21 +1,59 @@
...
+            names: data.names,
+            currentName: 0,
+            votingDone: false
+        }
+    }
+
+    componentDidMount() {
+        const id = parseInt(this.props.match.params.id);
+        const setPack = packData.packs.find(pack => pack.id === id);
+        this.setState({ pack: setPack});
```
Comment (EllieJellyBean) [inline src/Components/Pack/Pack.js:21]: Thanks for adding the mock data for now! This looks good!

**Review by EllieJellyBean — APPROVED** [review]
Looks great!

**Pull request merged**